design/high-level-reactivity.md

```diff
@@ -94,6 +100,31 @@ Before, we gave an example of a GraphQL response for a simple query. If you make
 }
 ```
 
+// I think following example is simple and complete.
+
+```
+{
+  me: {
+    username: "sashko",
+    lists: [
+      {
+        id: 1,
+        name: "My first todo list",
+      }
+    ],
+
+    __deps: {
+      _self: { key: '12341234', version: 3 },
+      lists: { key: '35232345', version: 4 }
+    }
+  }
+}
+```
+
+// Thinking about _self and lists. Is that because to avoid data re-fetching of lists when the username changes?
```

Comment:
Yes. So the idea is - if there is just the `_self` dep, it will refetch the whole subtree. If there is another one for a specific property, that will not be refetched. I think you have it right.

More details on this coming soon, we have a whole document about these deps, how they are returned, how they are refetched, etc. Hope to post that soon, maybe later this week, once I think about it some more, then we can discuss the details.